.github/workflows/e2e-tests.yml

```diff
@@ -145,6 +145,7 @@ jobs:
           export DfeSignIn__FrontDoorUrl=https://localhost:8081
           export GTM__Id=gtm_id
           export GTM__SiteVerificationId=testing_site_verification_id
+          export APP_ENVIRONMENT=E2E
```

Comment:
thats not `dev\test\staging\production` 👀

Reply:
can we reuse `ASPNETCORE_ENVIRONMENT` instead of a new thing?

Reply:
Can we reuse ASPNETCORE_ENVIRONMENT at all? which looks like it also needs to be one of the 4?

Reply:
I whoopsed and put that in the wrong place. I've fixed that now, but I'm not understanding what you mean :) Can you  explain please?

Reply:
I'm probably missing something obvious about the way cypress picks up the environment but why is it not `E2E` for all of them since it looks hardcoded?